Software Documentation/Naming and Coding Style Conventions.md

```diff
@@ -0,0 +1,48 @@
+### Variables
+- Constants
+    - Preceded by `k`
```

Comment:
Add a line about constants being static final members. This is because the fields says "Fields are any variables found in the class body." A new user may get confused as to which naming convention to use.